src/main/java/com/superdevs/HealthOMeter/dto/BMIDataDto.java

```diff
@@ -0,0 +1,77 @@
+package com.superdevs.HealthOMeter.dto;
+
+import com.superdevs.HealthOMeter.entity.Contact;
+
+import javax.persistence.Column;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+public class BMIDataDto {
+
+    private long id;
+    private BigDecimal height;
+    private BigDecimal weight;
+    private BigDecimal result;
+    private LocalDateTime created;
+    private Contact contact;
+
+    public BMIDataDto() {
+    }
+
+    public BMIDataDto(long id, BigDecimal height, BigDecimal weight, BigDecimal result, Contact contact) {
+        this.id = id;
```

Comment:
j.w.

Reply:
Fixing it right now, great suggestion, make sense.